src/components/badges/LicenseBadge.test.tsx

```diff
@@ -0,0 +1,83 @@
+/* @jsx MD */
+import MD, { render } from "jsx-md";
+import { Badge } from "../Badge";
+import { LicenseBadge } from "./LicenseBadge";
+
+describe("LicenseBadge", () => {
+  it("shows a license badge if repository is in npm shortform", () => {
+    const pkg = {
+      name: "package-name",
+      repository: "github:dbartholomae/jsx-readme",
+    };
+
+    expect(render(<LicenseBadge pkg={pkg} />)).toContain(
+      render(
+        <Badge
+          imageSource="https://img.shields.io/github/license/dbartholomae/jsx-readme"
+          link="https://raw.githubusercontent.com/dbartholomae/jsx-readme/main/LICENSE"
```

Comment:
Could you always link to the blob version instead of the CDN version of the files? E. g. this link shouln d be https://github.com/dbartholomae/jsx-readme/blob/main/LICENSE
GitHub offers additional UI especially around licenses.

Reply:
Noted, updated PR.